readme.md

```diff
@@ -117,7 +118,9 @@ function Status() {
 
 <a id="writabledrivedatom" href="#writabledrivedatom"><img src="img/rec.03.svg" alt="You can create a writable derived atom" /></a>
 
-Specify a write function at the second argument. `get` will return the current value of an atom, `set` will update an atoms value.
+Specify a write function at the second argument. 
+* `get` will return the current value of an atom
+* `set` will update an atoms value
```

Comment:
We strongly prefer to reduce lines in readme. Please revert it or suggest something within a line.

Reply:
Still working on this?

Reply:
that's it for now

Reply:
I mean, this specific comment. (I know you fixed everything except for this.)

> We strongly prefer to reduce lines in readme. Please revert it or suggest something within a line.

Reply:
Sorry I didn't realise I missed this. I've pushed a new change now.